Source/WebCore/platform/mac/ScrollbarsControllerMac.mm

```diff
@@ -1029,6 +1029,19 @@ - (void)invalidate
     [m_scrollerImpPair contentAreaScrolledInDirection:NSMakePoint(delta.width(), delta.height())];
 }
 
+void ScrollbarsControllerMac::scrollbarWidthChanged(WebCore::ScrollbarWidth)
+{
+    auto needsScrollerStyleUpdate = m_needsScrollerStyleUpdate;
```

Comment:
The cool kids are doing `auto needsScrollerStyleUpdate = std::exchange(m_needsScrollerStyleUpdate, false)`

Reply:
Actually this is more like a `SetForScope<>`. But why do we reset to false? A comment would help.

Reply:
This was originally to force a layout here, but I think it's actually not necessary. Removed.